docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+-----------------------------
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+You will need
+-------------
+
+1. Somewhere to install the Flocker client.
+Make sure you have **one** of the following on your machine:
+
+- OS X with `Homebrew <http://brew.sh/>`_ installed.
+- Ubuntu 14.04.
+
+2. Two hosts for two instances of Flocker.
+The options are:
+
+- Two Virtual Machines (VMs) on your local machine.
+  For this tutorial, you are supplied with Vagrant images to create the tutorial environment on VMs using VirtualBox, so you must have `Vagrant <https://www.vagrantup.com/>`_ and `VirtualBox <https://www.virtualbox.org/>`_ installed.
+- AWS or Rackspace (you will need an account with root access).
+- Physical hosts with a supported operating system.
+- Any combination of the above.
+
+.. note:: If you choose to use VMs on your local machine, you’ll need at least 4GB RAM.
+
+Contents
+--------
+
+.. contents:: 
+	:local:
+	:backlinks: none
+	:depth: 2
+
+Overview
+^^^^^^^^
+
+You will be controlling your Flocker cluster via the CLI you've installed locally.
+The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI.
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :width: 60 %
+   :alt: In the initial server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+The following diagram illustrates how the server-side Flocker setup will be configured at the end of the tutorial:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :width: 60 %
+   :alt: Following the completion of this tutorial the server-side Flocker setup will be configured with the web application still running within a container on the first server, while the Redis server with a volume is running on the second server.
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+.. To find out more about how Flocker manages migration of containers with volumes, see *add link here*
+
+Step 1: Installing the Flocker client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
```

Comment:
This splits the whole product into "The Flocker client" and "Flocker", but I thought that "Flocker" was the whole product. This is confusing.